src/js/components/Select/SelectContainer.js

```diff
@@ -428,11 +428,12 @@ class SelectContainer extends Component {
                         })
                       ) : (
                         <OptionBox
-                          align="start"
-                          pad="small"
+                          {...theme.select.container.box}
```

Comment:
OptionBox is per-option, but `theme.select.container` is the whole container, and `box` seems a bit generic as a key, since there are boxes everywhere. How about `theme.select.option.container` instead of `theme.select.container.box`?

Reply:
I like it. let me change it